### ISSUE TITLE
Windows is now broken

### ISSUE THREAD

**Issue by machenmusik** [nerfstudio-project/nerfstudio#1961]
```
    raise RuntimeError("Windows not yet supported for torch.compile")
RuntimeError: Windows not yet supported for torch.compile
```

**Comment by machenmusik** [comment]
The apparent breaking changes seem quite minor
https://github.com/nerfstudio-project/nerfstudio/pull/1948/files

A little surprised that support was removed entirely for both pytorch 1.12 and 1.13 without a transition period with deprecation warning

**Comment by machenmusik** [comment]
At present, the workaround seems to be
`git reset be29197`

**Comment by jkulhanek** [comment]
Hmm, https://github.com/pytorch/pytorch/issues/90768
I can for now disable torch.compile for windows.
the pr #1948 only dropped part of the code needed to support th1.13. I think it is not sufficient to undo that commit, is it? Also, _LRScheduler was renamed as LRScheduler

**Comment by machenmusik** [comment]
Thanks @jkulhanek.

> I think it is not sufficient to undo that commit, is it? Also, _LRScheduler was renamed as LRScheduler

Right, after I tried reverting that, I ran into the renaming.
I see you have already PR'd a backward compatibility fix, will give that a try over the weekend.

If eventually we plan to drop compatibility with Windows due to PyTorch limited support, we may want to signal that now; but from #1962 it seems at this point the changes are minor, and it may not yet be necessary.  

If torch.compile support makes Linux much more performant, we may want to put together a WSL2 guide for those on Windows, which might unblock the full transition to PyTorch 2 with torch.compile.

**Comment by machenmusik** [comment]
See https://github.com/nerfstudio-project/nerfstudio/pull/1962#pullrequestreview-1436741060

So with #1962 changes, WSL2 is not required to run  on Windows - but torch.compile does not work currently, which may be a performance disadvantage.

**Comment by machenmusik** [comment]
Fixed via #1962
